module_utils/helpers.py

```diff
@@ -488,6 +530,51 @@ def set_twophase_options_from_params(control_console, params):
     set_twophase_options(control_console, twophase_options)
 
 
+def read_yaml_file(control_console, file_path):
+    return control_console.eval_res_err(READ_YAML_FILE_FUNC_BODY, file_path)
+
+
+def get_clusterwide_config(control_console, filter_system=True):
+    current_config, err = control_console.eval_res_err('''
+        return require('cartridge').config_get_readonly()
+    ''')
+    if err is not None:
+        return None, err
+    if current_config is None:
+        return None, "Cluster isn't bootstrapped yet"
+
+    filtered = {}
+    for key, value in (current_config or {}).items():
+        if not key.endswith('.yml') and not (filter_system and key in SYSTEM_CONFIG_SECTIONS):
+            filtered[key] = value
+    return filtered, None
+
+
+def patch_clusterwide_config(control_console, new_sections):
+    current_config, err = get_clusterwide_config(control_console)
+    if err is not None:
+        return None, err
+
+    patch = {}
+    for section_name, section in new_sections.items():
+        if section_name in SYSTEM_CONFIG_SECTIONS:
+            return None, "Unable to patch config system section: '%s'" % section_name
+
+        if current_config.get(section_name) != section:
+            patch[section_name] = section
```

Comment:
```python
patch = {
    section_name: section
    for section_name, section in new_sections.items()
    if current_config.get(section_name) != section:
}
```

Reply:
We should to check that section is not system, so it's a not good variant for it.